packages/grid/_modules_/grid/components/GridViewport.tsx

```diff
@@ -60,6 +60,7 @@ export const GridViewport: ViewportType = React.forwardRef<HTMLDivElement, {}>(
           <GridRowCells
             columns={visibleColumns}
             row={r}
```

Comment:
could we cleanup here please? row?

Reply:
Fixed in https://github.com/mui-org/material-ui-x/pull/1377/files#diff-4d439fd92e7951c822fbe83c5b5357d461de18a795235b490c8c28230369f1f4R62.